main.js

```diff
@@ -137,29 +139,18 @@ function checkArrows() {
 
 function manageNextArrow(isNeeded) {
```

Comment:
I'm a fan of the refactoring for simplification! It seems to be working beautifully